proto/sharedstreets.proto

```diff
@@ -69,16 +69,16 @@ message SharedStreetsGeometry {
     string backReferenceId = 5;
 
     RoadClass roadClass = 6;
-    repeated float latlons = 7; // interleaved lat/lon pairs in sequence
+    repeated double lonlats = 7; // interleaved lon/lat pairs in sequence
```

Comment:
👍 `lonlats` is better than `latlons` especially if we want to translate these coordinates into the GeoJSON spec.

I'll make the adjustment downstream for the other Javascript libraries.